plextraktsync/sync.py

```diff
@@ -126,28 +126,37 @@ def trakt_wl_shows(self) -> Dict[int, TVShow]:
 
     def sync(self, walker: Walker, dry_run=False):
         listutil = TraktListUtil()
+        is_partial = walker.is_partial
 
         if self.config.update_plex_wl_as_pl:
-            listutil.addList(None, "Trakt Watchlist", trakt_list=self.trakt.watchlist_movies)
+            if is_partial:
+                logger.warning("Partial walk, disabling watchlist as playlist updating")
```

Comment:
```suggestion
                logger.warning("Running partial library sync. Watchlist as playlist won't update because it needs full library sync.")
```